### PR TITLE
Fix Edit Geometry button

### PR DESCRIPTION
**Pull request by vgeorge**
See #259. 

At this point the second item was implemented, component FeatureDetailsHeader was changed to not allow editing of feature type other that "Point" or "LineString".

### CODE REVIEW

File: app/components/FeatureDetailHeader.js [hunk 1]
```diff
@@ -88,16 +88,29 @@ export default class FeatureDetailHeader extends React.Component {
       navigation.navigate('SelectFeatureType', { feature })
     }
 
-    const featureType = feature.geometry.type === 'Point' ? 'node' : 'way'
+    // Get feature type to allow/disallow editing
+    let featureType
+    switch (feature.geometry.type) {
```
Comment (geohacker) [inline app/components/FeatureDetailHeader.js:93]: This is missing `Polygon`

**Comment by vgeorge** [comment]
Added two changes:

- Added Polygon as supported geometry 
- Refactored edit button to avoid showing it in AddFeatureDetail screen

File: app/components/FeatureDetailHeader.js [hunk 2]
```diff
@@ -63,6 +63,65 @@ export default class FeatureDetailHeader extends React.Component {
...
+      default:
+        featureType = 'unsupported'
+        break
+    }
+
+    if (featureType === 'unsupported') {
+      // If feature is unsupported, block further edits
+      return <Button>
+        <Coordinates>
+          <Edit>This feature was deleted</Edit>
```
Comment (sethvincent) [inline app/components/FeatureDetailHeader.js:106]: @vgeorge should this instead say something like `Editing this feature is not supported`?

Comment (vgeorge) [reply app/components/FeatureDetailHeader.js]: Of course. I updated it now.

**Pull request merged**